### PR TITLE
Update igc_lib.py

### PR DESCRIPTION
**Pull request by kuaka**
updated my fork to reflect changes in base. 

changes from last pull request: Updated degrees_float_to_degrees_minutes_seconds argument lon_or_lat to be not optional.

### CODE REVIEW

File: igc_lib.py [hunk 1]
```diff
@@ -58,28 +58,38 @@ def rawtime_float_to_hms(timef):
...
+    ddmmss = collections.namedtuple('ddmmss', ['hemisphere', 'degrees', 'minutes', 'seconds'])
     negative = dd < 0
     dd = abs(dd)
     minutes,seconds = divmod(dd*3600,60)
     degrees,minutes = divmod(minutes,60)
+    if lon_or_lat == 'lon':
+        hemisphere = 'E'
+    elif lon_or_lat == 'lat':
+        hemisphere = 'N'
+    else:
```
Comment (marcin-osowski) [inline igc_lib.py]: I guess you can remove this part now.

File: igc_lib.py [hunk 2]
```diff
@@ -58,28 +58,38 @@ def rawtime_float_to_hms(timef):
     
     return hms((time/3600), (time%3600)/60, time%60)
 
-def degrees_float_to_degrees_minutes_seconds(dd):
-    """Converts time from floating point degrees to degrees/minutes/floating point seconds.
+def degrees_float_to_degrees_minutes_seconds(dd, lon_or_lat):
```
Comment (marcin-osowski) [inline igc_lib.py:61]: Could you also update the `dump_thermals_to_cup_file()` function? It calls `degrees_float_to_degrees_minutes_seconds()`, and currently hard-codes "N" and "E".

File: igc_lib.py [hunk 3]
```diff
@@ -58,28 +58,38 @@ def rawtime_float_to_hms(timef):
...
+    """Converts from floating point degrees to degrees/minutes/floating point seconds.
     
     Args:
         dd: Floating point degrees to be converted
+        lon_or_lat: Optional string argument used to calculate the hemisphere.
```
Comment (marcin-osowski) [inline igc_lib.py]: Not optional any more.

**Comment by kuaka** [comment]
changes made to dump_thermals_to_cup_file(), degrees_float_to_degrees_minutes_seconds: docstring  and removed hemisphere of '?' if the lon_or_lat argument not 'lon' or 'lat' . Guess that will induce an error if a different value is used as the argument as function will try to return an un-initialised variable(hemisphere) .. probably not a bad thing.

**Pull request merged**